ansible/roles/netbox/defaults/main.yml

```diff
@@ -135,7 +135,7 @@ netbox__git_repo: 'https://github.com/netbox-community/netbox.git'
 # .. envvar:: netbox__git_version [[[
 #
 # The :command:`git` branch or tag which will be installed.
-netbox__git_version: 'v2.6.3'
+netbox__git_version: 'v2.8.7'
```

Comment:
You should also update the `meta/watch-netbox` watch file.

Since this PR was made, Netbox was updated to `v2.8.9`, perhaps you could use this instead. Or, we can wait for merging this PR for `v2.9.0` release, although I'm not sure when it will be released. I'm fine with switching to `v2.8.9` for now.

Reply:
Sounds good to me. Seems 2.8.9 fixed a bug I faced so I am motivated :) Let me test this for a few days.

Reply:
@ypid, the NetBox v2.9.1 has been released, do you want to try it out and update the PR for the current version?

Reply:
And again, v2.9.x fixes a few issues I faced. I just upgraded :) Lets see when I can finish the PR.